nsqlookupd/http.go

```diff
@@ -77,13 +77,40 @@ func (s *httpServer) doInfo(w http.ResponseWriter, req *http.Request, ps httprou
 	}, nil
 }
 
+type topicMeta struct {
+	Paused bool `json:"paused"`
+}
+
 func (s *httpServer) doTopics(w http.ResponseWriter, req *http.Request, ps httprouter.Params) (interface{}, error) {
+	reqParams, err := http_api.NewReqParams(req)
+	if err != nil {
+		return nil, http_api.Err{400, err.Error()}
+	}
+
+	topicName, _ := reqParams.Get("topic")
+	if topicName != "" {
+		var tm topicMeta
+		producers := s.ctx.nsqlookupd.DB.FindProducers("topic", topicName, "*")
+		producers = producers.FilterByActive(s.ctx.nsqlookupd.opts.InactiveProducerTimeout,
+			s.ctx.nsqlookupd.opts.TombstoneLifetime)
+		if producers.IsPaused() {
+			tm.Paused = true
+		}
+		return map[string]interface{}{
+			"topic_meta": tm,
```

Comment:
This new `topic` query param acts as a _filter_, so we should respond with the same fields and data either way.

Reply:
done.